### PR TITLE
Avatar Component

### PR DESCRIPTION
**Pull request by omerozkilic**
Hey!
Our avatar component has "type" prop. We have 3 type of avatar. Small - 32px, Normal - 44px and Large - 56px. I use "rem" instead of px. Our 1 rem value is coming from HTML - which is 16px.

### CODE REVIEW

File: packages/frontend/components/ui/avatar.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        avatarType[type],
+        'overflow-hidden rounded-full',
+        className
+      )}
+    >
+      <img
+        className="bg-no-repeat bg-center bg-contain"
+        src={image}
+        alt="noopener norefferer"
+      />
```
Comment (mustaphaturhan) [inline packages/frontend/components/ui/avatar.js]: @omerozkilic can you add here "what if there is no image" situation? what if I haven't add image yet?

Comment (omerozkilic) [reply packages/frontend/components/ui/avatar.js]: I never thought of it before. Should I do something like a placeholder for this?

Comment (omerozkilic) [reply packages/frontend/components/ui/avatar.js]: I commit this request to avatar branch.

File: packages/frontend/components/ui/avatar.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      )}
+    >
+      {image ? (
+        <img
+          className="bg-no-repeat bg-center bg-contain rounded-full"
+          src={image}
+          alt="noopener norefferer"
+        />
+      ) : (
+        <div className={cls(avatarType[type], 'bg-text-color', className)} />
```
Comment (mustaphaturhan) [inline packages/frontend/components/ui/avatar.js]: you're giving `avatarType[type]` class to parent div. why you're giving this class again? i guess it isn't necessary. you can use `w-full h-full` instead.

File: packages/frontend/components/ui/avatar.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    <div
+      className={cls(
+        avatarType[type],
+        'flex align-center justify-center overflow-hidden',
+        className
+      )}
+    >
+      {image ? (
+        <img
+          className="bg-no-repeat bg-center bg-contain rounded-full"
```
Comment (mustaphaturhan) [inline packages/frontend/components/ui/avatar.js]: are you sure about using `rounded-full` in here? i guess you can give `rounded-full` to parent div. img isn't the rounded one, container is. can you check this?

File: packages/frontend/stories/avatar.stories.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    />
+  ))
+  .add('Large Avatar', () => (
+    <Avatar
+      className={text('Classname')}
+      type="lg"
+      image="/img/placeholder-image-02.jpg"
+    />
+  ))
+  .add('Haven"t got Image Avatar', () => (
```
Comment (mustaphaturhan) [inline packages/frontend/stories/avatar.stories.js]: `'Haven"t...'` isn't right. you can use it like `"Haven't..."` but lets change it with something else that explain itself more.

my suggestion: `'Avatar without Image'`

**Comment by omerozkilic** [comment]
> thank you for changes. can you look for my notes?

Hey! sorry for this mistakes. I fixed this review items. Can you check again ?  Thanks.

File: packages/frontend/components/ui/avatar.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      )}
+    >
+      {image ? (
+        <img
+          className="bg-no-repeat bg-center bg-contain"
+          src={image}
+          alt="noopener norefferer"
+        />
+      ) : (
+        <div className="w-full h-full" />
```
Comment (mustaphaturhan) [inline packages/frontend/components/ui/avatar.js]:
```suggestion
        <div className="w-full h-full bg-text-color" />
```

**Pull request merged**